org.eclipse.jdt.ls.tests/src/org/eclipse/jdt/ls/core/internal/managers/InvisibleProjectImporterTest.java

```diff
@@ -217,7 +250,7 @@ public void testPreviewFeaturesDisabledForNotLatestJDK() throws Exception {
 			assertTrue(invisibleProject.exists());
 			assertNoErrors(invisibleProject);
 			IJavaProject javaProject = JavaCore.create(invisibleProject);
-			assertEquals(JavaCore.DISABLED, javaProject.getOption(JavaCore.COMPILER_PB_ENABLE_PREVIEW_FEATURES, false));
+			assertEquals(JavaCore.DISABLED, javaProject.getOption(JavaCore.COMPILER_PB_ENABLE_PREVIEW_FEATURES, true));
```

Comment:
why switching from false to true?

Reply:
It actually does fail with false because the result will be null. Interestingly, `javaProject.getOptions(false).get(JavaCore.COMPILER_PB_ENABLE_PREVIEW_FEATURES)` does return disabled though.

Weird.. The Javadoc mentions that getOption(String, boolean) is a `Helper method for returning one option value only. Equivalent to <code>(String)this.getOptions(inheritJavaCoreOptions).get(optionName)` :\

Reply:
Do you have any suggestions?